README.md

```diff
@@ -200,7 +200,8 @@ _See code: [src/commands/prune.ts](https://github.com/leapfrogtechnology/sync-db
 
 ## `sync-db synchronize`
 
-Synchronize all the configured database connections.
+Synchronize all the configured database connections. If `.dropped` is appended with an database object's name,
```

Comment:
@thesamiroli Can we just name it `.drop` instead of `.drop` because the sync-db synchronize will always try to drop the object even if it is not available.